README.md

```diff
@@ -130,9 +130,7 @@ Where:
      Use HTTP/2
 
    --timeout <uint32_t>
-     Timeout period in seconds used for both connection timeout and grace
-     period waiting for lagging responses to come in after the test run is
-     done. Default: 30.
+     Connection timeout period in seconds. Default: 30.
```

Comment:
Note: We had this concept of a timeout for the test waiting for outstanding requests to wrap up post-execution-duration.
This gets dropped here (the timeout will now be used solely as a connection timeout). This is done in this PR because it doesn't fit well with the termination predicate concept, and I think it also makes for easier reasoning about results. If we want something like this, then maybe we should think about introducing "phases" as a first-class citizen (e.g. warmup, cooldown, etc).

Reply:
Sure. But, as a courtesy to reviewers, really prefer that these unrelated things get cleaned up in a prelude PR.

Reply:
It is actually forwarded to `mutable_connect_timeout()` on the cluster. Updated.